hax/hax/util.py

```diff
@@ -693,7 +703,8 @@ def update_process_status(self, event: ConfHaProcess) -> None:
         assert 0 <= event.chp_event < len(ha_process_events), \
             f'Invalid event type: {event.chp_event}'
         local_node = self.get_local_nodename()
-        data = json.dumps({'state': ha_process_events[event.chp_event]})
+        data = json.dumps({'state': ha_process_events[event.chp_event],
+                           'type': str(m0HaProcessType(event.chp_type).name)})
```

Comment:
[Optional]: `str` looks redundant

Reply:
Here you change the type of value stored in Consul KV. Two comments:
1. Such a change should be reflected in 4/KV RFC.
2. Note that the value is not atomic. Are we sure that this is the final structure of the JSON and in the future versions we'll not want to change it again? I'm just afraid to introduce problems for upcoming SW upgrades.

Reply:
Will update 4/KV RFC.
I think its okay, as we have not released a kv version yet. We should avoid KV schema changes and try to identify and make changes as early as possible.